hackathon_site/hardware/test_api.py

```diff
@@ -49,10 +49,10 @@ def test_hardware_get_success(self):
         }
 
         response = self.client.get(self.view)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        data = response.json()
-
-        self.assertEqual(expected_response, data["results"][0])
+        # self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Why was this done? This test doesn't seem to be failing on my end

Reply:
my bad, just did it to test whether it works or not. Forgot to remove it.